src/plugins/discover/public/application/main/components/total_documents/total_documents.tsx

```diff
@@ -22,7 +22,12 @@ export const TotalDocuments = ({ totalHitCount }: { totalHitCount: number }) =>
         id="discover.docTable.totalDocuments"
         defaultMessage="{totalDocuments} documents"
         values={{
-          totalDocuments: <strong>{totalHitCount}</strong>,
+          totalDocuments: (
+            <strong>
+              {' '}
```

Comment:
One question, why did you add the `{' '}`?

Reply:
it was added on `linting`